README.rst

```diff
@@ -32,8 +32,9 @@ The result:
    +09deg 03' 23.1"
    2.33251 au
 
-Skyfield’s only binary dependency is NumPy.
-Once that is available, Skyfield can usually be installed with::
+Skyfield's only binary dependency is NumPy though jplepehm and sgp4
+are also required. Once those are available, Skyfield can usually be
```

Comment:
“Once those are available” — I think this phrase is misleading? It sounds like they need to be on the person's system already, which the don't, they merely need to be "available" on PyPI (which, as their author, I make sure they are). It sounds to me like this is likely to make people do extra work?

Reply:
Possibly my expectations were incorrect, but `python3 setup.py install` didn't pull down all the dependencies. Happy to change "available" to "installed".

Reply:
The command `setup.py install` in Python means, as far as I know, “installed this package but _not_ its dependencies” — so I think the result you saw was correct, and what you would see for any other package that has dependencies? The command to install including dependencies is `pip install .`.

Reply:
I'm sure that's what I get for not reading the docs super carefully every time I set up a new machine.

Roughly though, if I see a `requirements.txt` I `pip install -r` it, and then `setup.py` works. But I didn't see a `requirements.txt` so I didn't do anything special.

Reply:
Ah, yes, that in general won’t work. If you want to install a package from a checkout, `pip install .` installs the package and its dependencies. If you want to be able to edit files in the checkout and have them reflected in programs, `pip install -e .` will install the dependencies, then use symlinks to install the `.py` files so that edits to them in the repository are reflected in the next run of each Python program.

Glad we could get that straight for you!